php/PluginConfiguration.php

```diff
@@ -167,7 +168,11 @@ public static function isProcessingUnpublishedPosts(): bool
 
     public static function isNotProcessingUnpublishedPosts(): bool
     {
-        return get_option('coyote_skip_unpublished_enabled', true);
+        $skipUnpublishedEnabled = get_option('coyote_skip_unpublished_enabled', true);
+        if("on" == $skipUnpublishedEnabled || true == $skipUnpublishedEnabled)
```

Comment:
Should this not use `===` explicit comparison?

Could this not be simplified as `return "on" === $skipUnpublishedEnabled || true === $skipUnpublishedEnabled)`?